File: 0682-baseball-game/0682-baseball-game.cs
public class Solution {
    public int CalPoints(string[] operations) {
            List<int> nums = new List<int>();
        foreach( var operation in operations){
            if(operation == "C"){
                nums.RemoveAt(nums.Count-1);
            }
            else if(operation == "D"){
                nums.Add(nums[nums.Count-1]*2);
            }
            else if(operation == "+"){
                nums.Add(nums[nums.Count-1]+nums[nums.Count-2]);
            }
            else{
                nums.Add(int.Parse(operation));
            }
        }
        return nums.Sum();
    }
}

File: 0682-baseball-game/0682-baseball-game.java
class Solution {
    public int calPoints(String[] operations) {
        ArrayList<Integer> nums = new ArrayList<Integer>();
        int sum = 0;
        for(int i = 0;i < operations.length;i++){
            if(operations[i].equals("C")){
                nums.remove(nums.size()-1);
            }
            else if(operations[i].equals("D")){
                nums.add(nums.get(nums.size()-1)*2);
            }
            else if(operations[i].equals("+")){
                nums.add(nums.get(nums.size()-1)+nums.get(nums.size()-2));
            }
            else{
                nums.add(Integer.parseInt(operations[i]));
            }
        }
        for(int i=0;i<nums.size();i++){
            sum += nums.get(i);
        }
        return sum;
    }
}